src/01-functions.js

```diff
@@ -112,7 +124,14 @@ function averageThree() {}
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a,b) {
+  if (typeof(a)===typeof(b)){
```

Comment:
Let's watch our indentation here. - in the same way we have our code spaced out we need to make sure our code is nested properly.  We should indent one tab for each block that we nest in our code.